dash/include/dash/memory/GlobHeapMem.h

```diff
@@ -429,6 +429,23 @@ class GlobHeapMem
     return unit_local_size;
   }
 
```

Comment:
See my comments to `GlobPtr`. The same applies here. Maybe we can centralize the pointer arithmetic stuff.